packages/algob/src/internal/util/scripts-runner.ts

```diff
@@ -40,6 +40,11 @@ export async function runScript (
   runtimeEnv: AlgobRuntimeEnv,
   deployer: AlgobDeployer
 ): Promise<void> {
+  // if .ts file is encountered, load from /build/scripts/file.ts
+  if (relativeScriptPath.endsWith('.ts')) {
+    relativeScriptPath = path.join('build', relativeScriptPath.split('.ts')[0] + '.js');
+  }
```

Comment:
we are doing loading of js files here ?
why were we getting `undefined` error due to this ?

Reply:
Yes, as explained above we are loading the js files here (during run only), without changing the relative path in the beginning (in `loadFileNames`). 
No, we were not getting `undefined` because of this, this is kept here because of changed implementation. The other way (keeping the previous implementation) would be to handle loading from `/build` at all other places (as we're changing the script path in the deployer). Getting `undefined` bug was here - [https://github.com/scale-it/algo-builder/blob/7354b021f5a6424593e13cd4dfe591100b8e224c/packages/algob/src/builtin-tasks/run.ts#L94](https://github.com/scale-it/algo-builder/blob/7354b021f5a6424593e13cd4dfe591100b8e224c/packages/algob/src/builtin-tasks/run.ts#L94). We can fix the bug by updated the line above, but i think changing and handling path everywhere is not good, ad could maybe lead to more bugs. What do you think?

Reply:
yes, i think keeping it as single is less prone to bugs.